doc/src/try-plrust-with-docker.md

```diff
@@ -0,0 +1,74 @@
+# Try PL/Rust with Docker
+
+If you would like to give PL/Rust a try without succumbing to the long arduous process of installing everything that is required, then this next section is for you!
+
+The PL/Rust repository contains a Dockerfile named `Dockerfile.try` that contains everything you need to spin up and test PL/Rust in your own environment.
+
+The following instructions assume that you have a very basic understanding of what [Docker](https://www.docker.com) is and that you already have it installed. If you do not have Docker installed yet, instructions for your particular environment can be found here: <https://docs.docker.com/engine/install/>
```

Comment:
In general I think it's best if our docs don't address the reader directly.  So instead of saying, "... assume that you have a...", maybe something like "assume a very basic understanding of Docker".

There's a few instances of this below.  I suppose it's a style thing.  In the docs I just pushed I tried really hard to eliminate talking to the reader.